decentraland/AppNavigator.js

```diff
@@ -16,34 +16,64 @@ import EthereumQuestionScreen from "./screens/EthereumQuestionScreen";
 import EthereumSignUpScreen from "./screens/EthereumSignUpScreen";
 import EthereumSignInScreen from "./screens/EthereumSignInScreen";
 import MyAssetsScreen from "./screens/MyAssetsScreen";
+
 import Colors from "@constants/Colors.js";
+import { responsiveHeight } from "react-native-responsive-dimensions";
+
+const defaultNavigationOptions = {
+  headerStyle: {
+    backgroundColor: Colors.headerBackgroundColor,
+    height: responsiveHeight(7),
+  },
+};
 
-const defaultNavigationOptions = ({ navigation }) => {
+const headerWithMenuButton = ({ navigation }) => {
   return {
     headerLeft: (
       <Button transparent onPress={() => navigation.toggleDrawer()}>
         <Icon name="menu" />
       </Button>
     ),
-    headerStyle: {
-      backgroundColor: Colors.headerBackgroundColor,
-    },
-    headerTintColor: Colors.headerTintColor,
-    headerTitleStyle: {
-      fontWeight: "bold",
-    },
+  };
+};
+
+const headerWithBackButton = ({ navigation }) => {
```

Comment:
I wonder if we want to do an iOS/Android device check and not have the back button on Android?

Reply:
That's complicated so would come in a future PR if we went that route

Reply:
On Android Gmail app, the back button (arrow to left) is shown
![WhatsApp Image 2019-03-29 at 9 41 13 AM](https://user-images.githubusercontent.com/29674218/55223772-de0a2900-5206-11e9-843b-5f68ab24c19e.jpeg)

Reply:
Oh okay, cool. Thanks for checking that.

If that’s there on Android in that app then it shouldn’t be too jarring to Android users. Resolving this!